surah/src/commonMain/kotlin/com/islamversity/surah/model/UIItem.kt

```diff
@@ -12,7 +12,7 @@ data class AyaUIModel(
     val order: Long,
     val fontSize: Int,
     val translationFontSize: Int = 0,
-    val toolbarVisible: Boolean,
+    var toolbarVisible: Boolean,
```

Comment:
having mutable values in a concurrent world is not really a good option
if we set the value on each view then I think we don't need to have this property here at all